src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementProductTable.tsx

```diff
@@ -134,41 +134,43 @@ const ReimbursementProductTable: React.FC<ReimbursementProductTableProps> = ({
-                                  type="number"
-                                  autoComplete="off"
-                                  InputProps={{
-                                    startAdornment: <InputAdornment position="start">$</InputAdornment>
-                                  }}
-                                  onBlur={(e) => onCostBlurHandler(parseFloat(e.target.value), product.index)}
-                                  sx={{ width: '100%' }}
-                                  error={!!errors.reimbursementProducts?.[product.index]?.cost}
-                                />
-                              )}
-                            />
-                            <FormHelperText error>
-                              {errors.reimbursementProducts?.[product.index]?.cost?.message}
-                            </FormHelperText>
-                          </FormControl>
-                          <IconButton
-                            sx={{
-                              '&:focus': {
-                                backgroundColor: hoverColor
-                              }
-                            }}
-                            onClick={() => removeProduct(product.index)}
-                          >
-                            <Delete />
-                          </IconButton>
+                          <Box sx={{ marginTop: { xs: '6px', sm: '0px' }, display: 'flex', flexDirection: 'row' }}>
+                            <FormControl sx={{ width: '50%' }}>
+                              <Controller
+                                name={`reimbursementProducts.${product.index}.cost`}
+                                control={control}
+                                render={({ field }) => (
+                                  <TextField
+                                    {...field}
+                                    label={'Cost'}
+                                    size={'small'}
+                                    variant={'outlined'}
+                                    type="number"
+                                    autoComplete="off"
+                                    InputProps={{
+                                      startAdornment: <InputAdornment position="start">$</InputAdornment>
+                                    }}
+                                    onBlur={(e) => onCostBlurHandler(parseFloat(e.target.value), product.index)}
+                                    sx={{ width: '100%' }}
+                                    error={!!errors.reimbursementProducts?.[product.index]?.cost}
+                                  />
+                                )}
+                              />
+                              <FormHelperText error>
+                                {errors.reimbursementProducts?.[product.index]?.cost?.message}
+                              </FormHelperText>
+                            </FormControl>
+                            <IconButton
```

Comment:
Can you make it so this delete button is in the middle of description and cost and see how adding a bit more space between each description/cost stack looks.
<img width="740" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/1f37dfaf-2dc2-49eb-8641-e737283c94a6">

Reply:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/124754518/84d05bf4-9552-4f5b-8e95-91e3c0cb4a19)

@RChandler234 To clarify something like this?

Reply:
![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/124754518/0a740d99-4ce2-4ecb-811b-b4714149ca18)

![image](https://github.com/Northeastern-Electric-Racing/FinishLine/assets/124754518/0ac89657-e7c0-4077-95bc-ed85f8af94db)

Reply:
Yesh